src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -144,6 +149,26 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
             .toCompletableFuture()
             .join();
 
+    final CustomObjectDraft<JsonNode> customObjectDraft =
+        CustomObjectDraft.ofUnversionedUpsert(
+            RESOURCE_KEY, RESOURCE_KEY, JsonNodeFactory.instance.objectNode().put("name", "value"));
+    // following custom object should not be synced as it's created by the project-sync itself
+    final CustomObjectDraft<JsonNode> customObjectToIgnore =
+        CustomObjectDraft.ofUnversionedUpsert(
+            "commercetools-project-sync.runnerName.ProductSync.timestampGenerator",
+            "timestampGenerator",
+            JsonNodeFactory.instance.objectNode().put("name", "value"));
```

Comment:
The JsonNode object here can share with the object instantiated in line 154. No need to instantiate another object.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/168/commits/d9a1f4074eb30c8e136674b5c903ac45d40e46a3